Lab_15_Pig_Latin/functions.test.js

```diff
@@ -0,0 +1,25 @@
+const { translateToPigLatin } = require("./functions")
+
+describe("translateToPigLatin()", () => {
+    it(`takes in 'pickle' and returns 'icklepay'`, () => { // regular consonant
+        expect(translateToPigLatin("pickle")).toEqual("icklepay")
+    })
```

Comment:
```suggestion
describe("translateToPigLatin(str)", () => {
    it('adds ay to a word that starts with a regular consonant', () => {
        expect(translateToPigLatin("pickle")).toEqual("icklepay")
    })
```

Reply:
(In the `describe` block, remember to add the params for the function under test